tasks/certificates.yml

```diff
@@ -0,0 +1,41 @@
+---
+
+# TODO: Only request ticket when needed
```

Comment:
How about checking if a certain file (e.g. cert) exists and register a variable?